esp/esp/django_settings.py

```diff
@@ -242,7 +242,9 @@
 
 SESSION_EXPIRE_AT_BROWSER_CLOSE=True
 
-SESSION_ENGINE="django.contrib.sessions.backends.cached_db"
+SESSION_ENGINE = 'django.contrib.sessions.backends.cached_db' #which is persistent storage
+
+ATOMIC_REQUESTS = True
```

Comment:
I think if we're switching to `ATOMIC_REQUESTS` that deactivates `TransactionMiddleware`, so we may as well remove it now.  (The docs are unclear, so this is assuming that I read the [source](https://github.com/django/django/blob/stable/1.6.x/django/middleware/transaction.py) right.)